packages/core/src/search/search.ts

```diff
@@ -247,6 +252,7 @@ function parseKeyValue(searchRequest: SearchRequest, key: string, value: string)
     }
 
     case '_fields':
+    case '_elements':
```

Comment:
As mentioned earlier, let's break this out into 2 separate properties on `searchRequest`